ebos/eclgenericthresholdpressure.cc

```diff
@@ -133,6 +133,14 @@ finishInit()
                                maxRegions)));
     }
 
+    if (numEquilRegions_ > 2048) {
+        // warn about performance
+        OpmLog::warning(fmt::format("Number of equil regions is {}. This larger "
+                                 "than 2048. Note, that this might "
+                                 "might have a negative impact on performance "
+                                 "and memory consumption", numEquilRegions_));
+    }
```

Comment:
read read and fix fix this message message please please. I'd just remove the 'This larger than 2048'. While you chose that as the warning limit, there is nothing significant about the particular value as such.

Reply:
Thanks for the hint. Improved.